File: SecondYear/chaudharyaryanpanwar_AryanPanwar_2226cs1053_2/Week_1/forLoop.cpp
#include<bits/stdc++.h>
using namespace std;
int fibonnaci(int n){
        int a =1 ; 
        int b= 1 ;
        int sum = 0 ;
        if (n == 1) {
        return 1;
        }
        else if (n==2){
            return 2;
        }
        while (n-1>1) {
        sum = a+b;
        a = b;
        b =sum;
        // cout << sum << endl;
        n--;
        }
        return sum;
}
int main()
{
        /*
        *  Write your code here. 
         *  Read input as specified in the question.
         *  Print output as specified in the question.
        */
        int n;
         cin >> n;
         cout << fibonnaci(n);

}

File: SecondYear/Parth27904_ParthAgarwal_2226CSE(AI&ML)1015_2/Week_1/for_loop.java
import java.util.Scanner;

public class for_loop {


	public static void main(String[] args) {
		
		/* Your class should be named Solution.
	 	* Read input as specified in the question.
	 	* Print output as specified in the question.
		*/
		Scanner scanner  = new Scanner(System.in);
		int a = scanner.nextInt();
		int[] arr = new int[10000];
		arr[0] = 1;
		arr[1] = 1;
		for(int i = 2; i<a; i++){
			arr[i] = arr[i-1] + arr[i-2];
		}
		System.out.println(arr[a-1]);
        scanner.close();
	}

}
